cmd/operator-sdk/run/local.go

```diff
@@ -75,7 +76,11 @@ func (c runLocalArgs) runGo() error {
 	projutil.MustInProjectRoot()
 	absProjectPath := projutil.MustGetwd()
 	projectName := filepath.Base(absProjectPath)
-	outputBinName := filepath.Join(scaffold.BuildBinDir, projectName+"-local")
+	var exeSuffix string
+	if runtime.GOOS == "windows" {
+		exeSuffix = ".exe"
+	}
+	outputBinName := filepath.Join(scaffold.BuildBinDir, projectName+"-local"+exeSuffix)
 	if err := c.buildLocal(outputBinName); err != nil {
 		return fmt.Errorf("failed to build operator to run locally: %v", err)
```

Comment:
WDYT about we take this opportunity to add a few tests in the Travis for win platform as well?
@al45tair it is not a blocker to merge this PR, but in POV would be great if you see that it could be added without too much effort. 
@joelanford @estroz

Reply:
@camilamacedo86 I don't think we have _any_ tests around `run --local` or _any_ tests for Windows (correct me if I'm wrong). If not, I think it would be okay for this to merge without tests and for us to take an action to look into those things on the macro level.

Reply:
Yes, we have not. I agree with you.